terraform/anomaly_detection/main.tf

```diff
@@ -0,0 +1,213 @@
+  worker_disk_size_gb      = 200
+  machine_type             = "g2-standard-4"
+}
+
+
+// Project
+module "google_cloud_project" {
+  source          = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/project?ref=v32.0.0"
+  billing_account = var.billing_account
+  project_create  = var.project_create
+  name            = var.project_id
+  parent          = var.organization
+  services = [
+    "cloudbuild.googleapis.com",
+    "dataflow.googleapis.com",
+    "monitoring.googleapis.com",
+    "pubsub.googleapis.com",
+    "autoscaling.googleapis.com",
+    "artifactregistry.googleapis.com",
+    "bigtable.googleapis.com",
+    "bigtableadmin.googleapis.com",
+    "bigquery.googleapis.com",
+  ]
+}
+
+module "registry_docker" {
+  source     = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/artifact-registry?ref=v32.0.0"
+  project_id = module.google_cloud_project.project_id
+  location   = var.region
+  name       = "dataflow-containers"
+  iam = {
+    "roles/artifactregistry.admin" = [
+      "serviceAccount:${module.google_cloud_project.number}@cloudbuild.gserviceaccount.com"
+    ]
+    "roles/artifactregistry.reader" = [
+      module.dataflow_sa.iam_email
+    ]
+  }
+  cleanup_policy_dry_run = false
+  cleanup_policies = {
+    keep-3-versions = {
+      action = "KEEP"
+      most_recent_versions = {
+        keep_count = 3
+      }
+    }
+  }
+}
+
+// Buckets for staging data, scripts, etc, in the two regions
+module "buckets" {
+  source        = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/gcs?ref=v32.0.0"
+  project_id    = module.google_cloud_project.project_id
+  name          = module.google_cloud_project.project_id
+  location      = var.region
+  storage_class = "STANDARD"
+  force_destroy = var.destroy_all_resources
+}
+
+module "input_topic" {
+  source     = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/pubsub?ref=v32.0.0"
+  project_id = module.google_cloud_project.project_id
+  name       = "messages"
+  subscriptions = {
+    messages-sub = {}
+  }
+}
+
+module "output_topic" {
+  source     = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/pubsub?ref=v32.0.0"
+  project_id = module.google_cloud_project.project_id
+  name       = "predictions"
+  subscriptions = {
+    predictions-sub = {}
+  }
+}
+
+//bigtable table
+module "enrichment_table" {
+  source     = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/bigtable-instance?ref=v32.0.0"
+  project_id = module.google_cloud_project.project_id
+  name       = "bt-enrichment"
+  clusters = {
+    cluster1 = {
+      zone      = var.zone
+      num_nodes = 3
+    }
+  }
+  tables = {
+    features = {}
+  }
+}
+
+//bigquery dataset
+module "output_dataset" {
+  source     = "github.com/GoogleCloudPlatform/cloud-foundation-fabric//modules/bigquery-dataset?ref=v32.0.0"
+  project_id = module.google_cloud_project.project_id
+  id         = "output_dataset"
```

Comment:
This could be a local variable, so it is easier to find in the code in case someone wants to change it.